### PR TITLE
[SERVICE-444] Add Service Tooling Testing Integration

### PR DESCRIPTION
**Pull request by ryanpetersonOF**
Lets service-tooling handle the startup of Jest unit/int tests.

### CODE REVIEW

File: test/demo/tabbing/createTabFromMultipleWindows.inttest.ts [hunk 1]
```diff
@@ -17,9 +17,9 @@ afterEach(async () => {
     await win1.close();
     await win2.close();
     fin.InterApplicationBus.removeAllListeners();
-});
 
-afterEach(teardown);
```
Comment (bryangaleOF) [inline test/demo/tabbing/createTabFromMultipleWindows.inttest.ts]: Does this make a difference or is this just a style change?

File: test/demo/workspaces/basicSaveAndRestore.inttest.ts [hunk 2]
```diff
@@ -27,8 +27,10 @@ numberOfApps.forEach(appNumber => {
     });
 });
 
-afterEach(closeAllPreviews);
-afterEach(teardown);
+afterEach(async ()=>{
```
Comment (bryangaleOF) [inline test/demo/workspaces/basicSaveAndRestore.inttest.ts]: Would format as `async () => {` to match elsewhere. Ditto for the other `afterEach(async ()=>{`s

File: test/demo/tabbing/createTabFromMultipleWindows.inttest.ts (hunk 1 above)
Comment (ryanpetersonOF) [reply test/demo/tabbing/createTabFromMultipleWindows.inttest.ts]: The two `afterEach` were causing issues so I put them all into one.

File: test/demo/workspaces/basicSaveAndRestore.inttest.ts (hunk 2 above)
Comment (tomrobertsOF) [reply test/demo/workspaces/basicSaveAndRestore.inttest.ts]: I would have thought the linter would catch that?

Comment (ryanpetersonOF) [reply test/demo/workspaces/basicSaveAndRestore.inttest.ts]: Good catch

**Comment by ryanpetersonOF** [comment]
Just waiting for tests to pass on this

**Pull request merged**